build.gradle

```diff
@@ -3381,16 +3388,21 @@ project(":web") {
 
     test {
         doFirst {
-            if (!IS_COMPILE_WEBKIT) {
-                println "*****************************************************"
+            if (IS_STUB_RUNTIME_OPENJFX) {
```

Comment:
Perhaps this should be inside the `if (!IS_COMPILE_WEBKIT)` ?

Reply:
I assume you mean like this:
````
            if (!IS_COMPILE_WEBKIT) {
                if (IS_STUB_RUNTIME_OPENJFX) {
                   println ...
                } else {
                   println ...
		}
            }
````
I do not like the additional indentation level, so I have left it as is.

The semantics should be the same, as long as no one use stubs *and* compile webkit.
Which should maybe rather cause a build exception?

But it is not something I feel strongly for, so I am happy to change it :)

Reply:
Yes, that's what I was suggesting, but I'm happy with the way it is.